### PR TITLE
Personal Portfolio

### PR DESCRIPTION
**Pull request by Sheikh-Umar**

### CODE REVIEW

File: docs/team/Sheikh-Umar.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+* *Code contributed*: [https://github.com/CS2103JAN2018-F11-B1/main/blob/master/collated/functional/Sheikh-Umar.md[Functional code]] [https://github.com/CS2103JAN2018-F11-B1/main/blob/master/collated/test/Sheikh-Umar.md[Test code]]
+
+* *Other contributions*:
+
+** Project management:
+*** Managed the collation of code of my team for releases `v1.4` - `v1.5rc` (2 releases) on GitHub
+** Documentation:
+*** Did cosmetic tweaks to existing contents of the User Guide: https://github.com[#14]
+** Community:
+*** PRs reviewed (with non-trivial review comments): https://github.com/CS2103JAN2018-F11-B1/main/pull/158[#158], https://github.com/CS2103JAN2018-F11-B1/main/pull/159[#159]
```
Comment (davidten) [inline docs/team/Sheikh-Umar.adoc]: You didn't even leave a comment when reviewing these

File: docs/team/Sheikh-Umar.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+*** Contributed to forum discussions (examples:  https://github.com[1], https://github.com[2], https://github.com[3], https://github.com[4])
```
Comment (davidten) [inline docs/team/Sheikh-Umar.adoc]: The 4 links are to github.com.

File: docs/team/Sheikh-Umar.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+*** Reported bugs and suggestions for other teams in the class (examples:  https://github.com[1], https://github.com[2], https://github.com[3])
```
Comment (davidten) [inline docs/team/Sheikh-Umar.adoc]: These links are to github.com

File: docs/team/Sheikh-Umar.adoc [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+== Contributions to the User Guide
+
+|===
+|_Given below are sections I contributed to the User Guide. They showcase my ability to write documentation targeting end-users._
+|===
+
+include::../UserGuide.adoc[tag=display]
```
Comment (davidten) [inline docs/team/Sheikh-Umar.adoc]: This links to the entire user guide

File: docs/team/Sheikh-Umar.adoc [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+== Contributions to the Developer Guide
+
+|===
+|_Given below are sections I contributed to the Developer Guide. They showcase my ability to write technical documentation and the technical depth of my contributions to the project._
+|===
+
+include::../DeveloperGuide.adoc[tag=alias]
```
Comment (davidten) [inline docs/team/Sheikh-Umar.adoc]: These links to the entire dev guide

File: docs/team/Sheikh-Umar.adoc (hunk 1 above)
Comment (Sheikh-Umar) [reply docs/team/Sheikh-Umar.adoc]: I thought approving the PR was enough. Will make necessary changes.

File: docs/team/johndoe.adoc [hunk 6]
```diff
@@ -1,71 +0,0 @@
-= John Doe - Project Portfolio
```
[the remark this answers is not in the document]
Comment (Sheikh-Umar) [reply docs/team/johndoe.adoc]: Ok. I'll add it back in.

File: docs/team/Sheikh-Umar.adoc (hunk 5 above)
Comment (Sheikh-Umar) [reply docs/team/Sheikh-Umar.adoc]: This shows only the portion of the DeveloperGuide on the alias command.

Comment (davidten) [reply docs/team/Sheikh-Umar.adoc]: https://github.com/Sheikh-Umar/main/blob/9e6343360b49325a1811695b0bfa5de8a2274331/docs/team/Sheikh-Umar.adoc

Go click on your link it goes to the full dev guide

Comment (Sheikh-Umar) [reply docs/team/Sheikh-Umar.adoc]: Ok now I get it.

**Comment by Sheikh-Umar** [comment]
I've edited the links to my portions of the DeveloperGuide. Unsure if it should be done like this though.

**Comment by davidten** [comment]
You really need to go open the portfolio and take a look at how it looks before sending it back for review

**Comment by davidten** [comment]
@Sheikh-Umar let me know when you finished working on this, your dev guide link still links to the entire dev guide

**Comment by Sheikh-Umar** [comment]
Noted. I'll keep this PR opened, and comment here once I've finished working on it.

**Comment by Sheikh-Umar** [comment]
@davidten You may review my PR.

**Pull request merged**